README.md

```diff
@@ -46,6 +44,9 @@ A [Terraform] 0.12 module for managing an organization on [Github].
 
 ## Getting Started
 
+The GitHub Provider requires a GitHub Token and the name of the organization.
+You can provide your credentials via the `GITHUB_TOKEN` and `GITHUB_ORGANIZATION` environment variables.
+
```

Comment:
this is not the scope of the module to care about provider configuration.. as people can use all the ways to configure this when they define the provider.. so if the provider picks up the same config of gh-cli this is even not needed at all.

Reply:
resolved in c2d1bab